proto/voice-info.proto

```diff
@@ -0,0 +1,175 @@
+syntax = "proto3";
+
+package org.sim.voices;
+
+/*
+  Version info, shall adhere to semantic versioning in Hex number for easier
+  comparison.  E.g. 0x00010001 for version 1.0.1.
+  Rules are as follows:
+    - Major number shall be increased, in case of incompatible changes,
+        e.g. semantically incompatible change, new parameters needed, etc.
+    - Minor number shall be increased, in case of a compatible changes,
+        e.g. renaming of members, adding new members, changing the protobuf
+        type in a compatible way, improving a models speed/quality, etc.
+    - Patch number shall be increased, in case of a backwards-compatible
+      bug-fix
+*/
+message Revision {
+  uint32 version = 1;
+}
+
+/* Describes a voice model's file entry */
+message FileEntry {
+  enum Compression {
+    COMPRESSION_PLAIN = 0;      // no compression
+    COMPRESSION_BROTLI = 1;
+    COMPRESSION_BZIP2 = 2;
+    COMPRESSION_GZIP = 3;
+    COMPRESSION_ZIP = 4;
+  }
+  enum Format {
+    FORMAT_PLAIN = 0;           // plain file, no archive
+    FORMAT_TAR = 1;             // tar archive
+    FORMAT_CPIO = 2;            // cpio archive
+  }
+  string path = 1;                // relative path, mandatory
+  Compression compression = 2;    // file compression, optional
+  Format format = 3;              // file format, optional
+  string md5 = 4;                 // MD5 sum of file, mandatory
+}
+
+/* Speech model language / locale info */
+message Language {
+  /* <ISO 639 language code>_<ISO 3166 country code> */
+  enum Locale {
+    /* default: any language/country) */
+    LOCALE_ANY = 0;
+    is_IS = 1;
+    /* more entries can be appended here ... */
+  }
+  Locale locale = 1;  // locale of the voice, mandatory
+  string name = 2;    // language name of the voice, mandatory
+}
+
+/* Phonemes used for acoustic model, alphabets and symbol mappings */
+message PhonemeDescription {
+  enum PhonemeType {
```

Comment:
Do we have an URL I could refer to for the symbols used for IPA/X-Sampa ?

Reply:
Our central document regarding the alphabets is the pdf from Eiríkur Rögnvaldss., we need to deliver the lists and mappings with each repository using these alphabets. But a URL would be very useful, I think, where these lists and mappings would be available as .txt. There is no "international" URL for this, if that is what you mean

Reply:
Is this PDF part of Clarin or is there a public repo for it ? Otherwise, I could simply attach the PDF inside this Repo

Reply:
It is available on CLARIN as a separate deliverable, but is also included in the g2p repositories. I would also propose to include a more accessible version with direct mappings in plain format - which is also a part of the g2p repo